src/extension.ts

```diff
@@ -159,4 +159,7 @@ async function startLanguageServer() {
             g_serverstatus.hide();
         })
     })
+    vscode.workspace.onDidChangeWorkspaceFolders((e) => {
+        g_languageClient.sendRequest('workspace/didChangeWorkspaceFolders', e.added)
+    })

```

Comment:
Don't we have to handle removing workspace folders as well?

Reply:
Oh, and I kind of had assumed the languageClient would handle this stuff automatically? Seems weird that we need to listen to the ``onDidChangeWorkspaceFolder`` event here and then rerout it to the LS...

Reply:
It's supposed to automatically send a notification on the above event when both server and client have `workspaceFolders`cababilities but doesn't in practice